src/openzaak/templates/admin/catalogi/publish_zaaktype.html

```diff
@@ -0,0 +1,58 @@
+{% extends "admin/index.html" %}
+{% comment %} SPDX-License-Identifier: EUPL-1.2 {% endcomment %}
+{% comment %} Copyright (C) 2019 - 2024 Dimpact {% endcomment %}
+{% load i18n %}
+
+{% block title %} {% trans "Import zaaktype" %} {{ block.super }} {% endblock %}
+
+{% block content_title %}{% endblock %}
+{% block sidebar %}{% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; <a href="{% url 'admin:catalogi_zaaktype_changelist' %}">{% trans 'Zaaktype' %}</a>
+&rsaquo; {% trans 'Publiceren zaaktype' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans 'Publiceren zaaktype' %}</h1>
+
+    <h2>{{ zaaktype }}</h2>
+
+{% if zaaktype.informatieobjecttypen.all %}
+    <table class="module aligned ">
+        <tr><th>{% trans 'Informatieobjecttypen' %}</th> <th>{% trans 'Gepubliceerd' %}</th></tr>
+        {% for iot in zaaktype.informatieobjecttypen.all %}
+        <tr><td>{{iot}}</td> <td>{{iot.is_published | yesno}}</td></tr>
```

Comment:
Should it be displayed in the same way like in list view? Red cross or green checkmark? 
This comment is not critical. If it's too much work you can skip it